packages/wrangler/src/cloudchamber/client/models/AccountDefaults.ts

```diff
@@ -9,5 +9,10 @@ import type { MemorySizeWithUnit } from "./MemorySizeWithUnit";
  */
 export type AccountDefaults = {
 	vcpus: number;
+	memory_mib?: number;
+	/**
+	 * DEPRECATED
+	 * @deprecated
```

Comment:
Could the deprecation (here and for the rest of the files) include a reason for the deprecation and/or what alternative to use?